cronjob.py

```diff
@@ -0,0 +1,81 @@
+
+from lib.modules.networkingmodule.networkingresponse import *
+import requests
+import json
+import numpy as np
+import os
+from dotenv import load_dotenv
+from pathlib import Path
+from aiohttp import ClientError
+from lib.modules.databasemodule.database import database
+db = database.instance()
+
+env_path = Path('.') / '.env'
+load_dotenv(dotenv_path=env_path)
+
+if __name__ == '__main__':
```

Comment:
this is incredibly long, can you break it up into several different functions so it is easier to read